client/src/components/categorical/value/index.js

```diff
@@ -739,7 +739,7 @@ class CategoryValue extends React.Component {
                       ) : null}
                       {displayString !== globals.unassignedCategoryLabel ? (
                         <MenuItem
-                          icon="delete"
+                          icon="trash"
                           intent="danger"
```

Comment:
should this be `Intent.DANGER`?  (ditto for "trash")

Reply:
https://github.com/chanzuckerberg/cellxgene/issues/2120